flake.nix

```diff
@@ -6,7 +6,7 @@
 
     flake-utils.url = "github:numtide/flake-utils";
 
-    nix-update-flake.url = "github:Mic92/nix-update";
+    nix-update-flake.url = "github:Mic92/nix-update/09aadb5d6d9e1fc57df0b61def4bdd8b43ea47a1";
```

Comment:
The weekly updates seem to work. How is this recipe different?

Reply:
Okay, I triggered the daily workflow manually, and it seems like it worked: https://github.com/dfinity/motoko/actions/runs/17547142050/job/49831108455